ovos_stt_plugin_fasterwhisper/__init__.py

```diff
@@ -1,19 +1,20 @@
 import numpy as np
-from faster_whisper import WhisperModel, decode_audio
+from faster_whisper import WhisperModel, decode_audio, available_models
 from ovos_plugin_manager.templates.stt import STT
 from ovos_plugin_manager.templates.transformers import AudioLanguageDetector
 from speech_recognition import AudioData
-
+from ovos_utils.log import LOG
 
 class FasterWhisperLangClassifier(AudioLanguageDetector):
     def __init__(self, config=None):
         config = config or {}
         super().__init__("ovos-audio-transformer-plugin-fasterwhisper", 10, config)
         model = self.config.get("model")
-        if not model:
+        valid_model = model in FasterWhisperSTT.MODELS
+        if not model or not valid_model:
+            LOG.warning(f"{model} is not a valid model ({FasterWhisperSTT.MODELS}), using 'small' instead")
```

Comment:
can we log the valid options here too?

Reply:
That's what `FasterWhisperSTT.MODELS` is

Reply:
Output from test:

```py
2024-05-23 12:12:48.361 - OVOS - ovos_stt_plugin_fasterwhisper:__init__:15 - WARNING - invalid_model is not a valid model (['tiny.en', 'tiny', 'base.en', 'base', 'small.en', 'small', 'medium.en', 'medium', 'large-v1', 'large-v2', 'large-v3', 'large', 'distil-large-v2', 'distil-medium.en', 'distil-small.en', 'distil-large-v3']), using 'small' instead
```